requirements.txt

```diff
@@ -1,37 +1,126 @@
+aiohttp==3.6.3
+amqp==5.0.9
```

Comment:
I had been just including the top-level dependencies here (and all sub-dependencies will get automatically installed). But sounds like including everything from `pip freeze` is common and has some benefits. I don't do enough Python development to know what the best practice is, so either way works with me.

Reply:
including everything is my hunch, in line with committing `package-lock` in node projects.